### DISCUSSION TITLE
Selecting custom fields

### DISCUSSION THREAD

**Discussion by ConstructionGamer** [Courseplay/courseplay#7703] [General]
When creating custom fields is it possible to select a custom field which is within an existing field boundary? If I create a custom field within an existing field CP seems to default to the existing field. Or is there another way to select a field other than using "field position". I'm just thinking about those who may want to split 1 field into 2.

**Comment by Tensuko** [comment]
Global settings, switch the selecting prority. And you are in the wrong repository...

**Reply by ConstructionGamer** [reply]
Thank you 🙂. Sorry for posting in wrong repository.
